ddp-workspace/projects/toolkit/src/lib/components/dashboard/dashboard.component.ts

```diff
@@ -89,11 +94,18 @@ export class DashboardComponent implements OnInit, OnDestroy {
         this.announcementMessages[index].shown = false;
     }
 
-    public navigate(id: string): void {
+    public navigate(id: string, participantGuid?: string): void {
+        if (this.useParticipantDashboard) {
+            this.session.setParticipant(participantGuid);
```

Comment:
the current implementation of activity services allows navigating through activities of another user (participant). It's implemented by using participant guid from the session object. It was already used in atcp for example and in the prism.
We set the session participant guid before navigating in order to use already implemented mechanisms.
Not sure what we should rename, could you please be more specific? participantGuid?

Reply:
I see your point here. I think it's fine that setParticipant takes id only. Probably the point of the method name setParticipant 
 is about that we update the session object as it may behaves from participant point of view, Right now we only need guid for that. It might need more data later